rollup.config.js

```diff
@@ -30,15 +28,8 @@ const conf = {
   ]
 }
 
-if (buildTest) {
-  conf.output.file = './tests/dist/js/bs-stepper.js'
-}
-
 if (buildDev) {
   conf.output.file = './tests/dist/js/bs-stepper.js'
-  conf.watch = {
```

Comment:
watch is still working in dev ?

Reply:
I remember it didn't work without this patch, but please try it yourself too.